terraform/gcp/k8s/templates/ingress.yml

```diff
@@ -13,7 +13,7 @@ metadata:
   name: juniper-ingress
   annotations:
     kubernetes.io/ingress.global-static-ip-name: admin-ip
-    networking.gke.io/managed-certificates: "admin-cert,{{ template "customer-certs" . }}"
+    networking.gke.io/managed-certificates: "{{ .Values.adminCertName }},{{ template "customer-certs" . }}"
```

Comment:
I forgot to update the reference to the cert name, so I had to update it manually

In retrospect: I don't think this was actually necessary. I think it _can_ regen in place, _but_ I had to manually clean up a bunch of partially deleted certs that were lingering on the ingress (but did not exist) - possible caused by race condition from deleting a bunch all at once?

I think if I had fixed that issue first, this admin-renaming thing wouldn't have been necessary.